resources/backup-init/values.yaml

```diff
@@ -2,5 +2,5 @@
 kubectl:
   image:
     repository: eu.gcr.io/kyma-project/pr/etcd-tls-setup
```

Comment:
such image `eu.gcr.io/kyma-project/pr/etcd-tls-setup:a12c148d` does not exist. Why CI pipelines didn't catch that?